posixlib/src/main/resources/scala-native/unistd.c

```diff
@@ -26,12 +26,40 @@
  * prevent an application developer from getting the parts of an application
  * which do not actually use the constants from running.
  */
+
+#ifdef _XOPEN_VERSION
+#undef _XOPEN_VERSION
+#endif
 #define _XOPEN_VERSION 0
```

Comment:
I guess it would be better to keep original macro
```suggestion
#ifndef _XOPEN_VERSION
#define _XOPEN_VERSION 0
#endif
```

Reply:
Not sure, because only part of constant are missed and all of them is switched of.

Let me ping @LeeTibbert as well.